Resources/Prototypes/_Goobstation/Heretic/Entities/Objects/magic_contraband.yml

```
@@ -0,0 +1,11 @@
+- type: entity
+  id: BaseMagicContraband
+  abstract: true
+  components:
+  - type: Contraband
+    severity: Magic
+    allowedDepartments: null
+
+- type: contrabandSeverity
+  id: Magic
+  examineText: contraband-examine-text-magic
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)